portfolio/src/main/webapp/script.js

```diff
@@ -17,25 +17,30 @@
  */
 function addRandomGreeting() {
   var greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  var greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
-function addRandomIntro() {
-  var intros =
-      ['Student', 'Developer', 'Woman', 'Human'];
+  ['Hello world! I\'m a...', '¡Hola Mundo! Yo soy...', '你好，世界！我是...', 'Bonjour le monde! Je suis...'];
+    var greeting = greetings[Math.floor(Math.random() * greetings.length)];
```

Comment:
In general, please use `let` (I was mistaken to use var). 

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let
"var keyword must not be used." (for local scopes like functions and such)